pkg/multiplexer/multiplexer.go

```diff
@@ -39,6 +39,24 @@ func newTimeoutError(inner error) *TimeoutError {
 	return &TimeoutError{inner: inner}
 }
 
+type CancellationError struct {
```

Comment:
```suggestion
type ErrRequestCanceled struct {
```

Can we update this error to use the `Err` prefix convention?

Reply:
done in 4fe5e294cd2f1f09e00b72579315bd918c44d66b